test/index.ts

```diff
@@ -13,3 +18,40 @@ describe('Hello test', () => {
     expect(res.body.data.hello).to.be.eq('Hello, world!');
   });
 });
+
+describe('createUser test', () => {
+  it('createUser mutation', async () => {
+    const res = await request('localhost:4000')
+      .post('/graphql')
+      .send({
+        query: `
+        mutation {
+          createUser(
+            data: { name: "Marco", email: "marco@email.com", password: "pswd123", birthDate: "04-01-2000" }
+          ) {
+            id
+            name
+            email
+            birthDate
+          }
+        }
+      `,
+      });
+    expect(res.body.data.createUser.id).to.exist;
+    expect(res.body.data.createUser.name).to.be.eq('Marco');
+    expect(res.body.data.createUser.email).to.be.eq('marco@email.com');
+    expect(res.body.data.createUser.birthDate).to.be.eq('954558000000');
+
+    const userRepository = getConnection().getRepository(User);
+    const dbUser = await userRepository.findOne(res.body.data.createUser.id);
+    expect(dbUser.name).to.be.eq('Marco');
+    expect(dbUser.email).to.be.eq('marco@email.com');
+    expect(dbUser.birthDate.toDateString()).to.be.eq(new Date('2000-04-01T03:00:00.000Z').toDateString());
+    expect(dbUser.password).to.not.be.eq('pswd123');
```

Comment:
agora que to vendo, essa parte não me parece correta. A senha não deve ser salva no banco em plain text: https://github.com/indigotech/onboard-marco-prado/issues/10 (item 4), e inclusive ta ai mais um cenario de teste

Reply:
Coloquei esse teste para comparar a senha do banco com a senha original, para ver se a que está no banco foi hasheada, mas pensando melhor agora, parece não ser uma ideia muito boa fazer desse jeito... vou fazer essa checagem de outra maneira

Reply:
ahh, agora vi o `not`. Sorry :shame:
Então... uma sugestão seria vc fazer o hash da senha que ta no input e ver se bate com o que ta no banco